shared/config.lua

```diff
@@ -1128,3 +1128,79 @@ Config.InitialPlayerClothes = {
         }
     }
 }
+
+Config.DefaultSkin = {

```

Comment:
This is unnecessary and duplicated code. Extend `Config.InitialPlayerClothes` instead and add the missing parts there in lua table format, not indexed string format

Reply:
> This is unnecessary and duplicated code. Extend `Config.InitialPlayerClothes` instead and add the missing parts there in lua table format, not indexed string format

already revert